README.md

```diff
@@ -96,9 +96,9 @@ end
 ### jenkins_credentials
 **NOTE** The use of the Jenkins credentials resource requries the Jenkins credentials plugin. This plugin began shipping with Jenkins 1.536. On older Jenkins installations, you will need to install the credentials plugin at version 1.5 or higher to utilize this resource. On newer versions of Jenkins, this resource should work correctly.
 
-Each credential can be referenced in job by its UUID.
-You can set this UUID when creating credential, and set the same UUID in job configuration.
-To generate UUID, you can use linux command `uuidgen`.
+Each credential can be referenced in job by its unique ID.
+You can set this ID when creating credential, and set the same ID in job configuration.
+To generate a unique ID, you can use linux command `uuidgen`.
```

Comment:
The ID has been introduced in Jenkins Credentials Plugin 1.5 and has been a string ever since, see https://github.com/jenkinsci/credentials-plugin/commit/019b4a17b3bcaaf1547823699742440f68d742b4#diff-f2130afb4718830431143922c4d72ae4R42.

Internally, when creating new credentials, the plugin generates a UUID and converts it to a string. When using the API, any unique string can be used when creating credentials. With [JENKINS-26099](https://issues.jenkins-ci.org/browse/JENKINS-26099) it's now possible to set the ID from the web UI to any unique value. But when omitting the ID, a UUID will be generated.